config/resourcefulbees/bees/special/Soup.json

```diff
@@ -21,7 +21,7 @@
         "mutations": [
             {
                 "type": "ITEM",
-                "inputID": "minecraft:bowl",
+                "inputID": "minecraft:oak_log",
```

Comment:
Is this change intended? The bowl made a bit more sense imo.

Reply:
It totally does, and currently isn't supported unfortunately. That's why the recipe was failing to load - you can mutate TO an item, but not from one. Closest thing to a bowl I could come up with was a block of wood... I would change this if/when Epic implements item mutations.